fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -100,6 +102,31 @@ public void setDependencies(@Nonnull Descriptors.FileDescriptor[] dependencies)
         this.dependencies = dependencies;
     }
 
+    /**
+     * Get local meta-data file descriptor.
```

Comment:
```suggestion
     * Gets the local meta-data file descriptor.
```

It probably could use a link to `RecordMetaDataBuilder.setLocalFileDescriptor`.

Reply:
I believe the rest of the getters/setters in this file do not have an 's' at the end of Get and Set.  I prefer to continue following the way others are. Sure, added the link.

Reply:
Yeah, I can live without the "s". It should have a "the", though.

Reply:
that's fixed. thank you for the review.